permission_prompt/security_privacy_self_review.md

```diff
@@ -0,0 +1,108 @@
+# Security and Privacy Self-Review: Private Network Access Permission Prompt
+
+Based on the W3C TAG's
+[questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/).
+
+## 1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+Documents might be able to infer whether they were loaded from `public`,
+`private` or `local` IP address by observing whether they can successfully load
+a well-known subresource from the private network or localhost. This is largely
+unavoidable by design: this specification aims to alter user agent behavior
+differentially based on the IP address which served the main resource.
+
+This can also arise when a client loaded a resource over a non-public proxy,
+such as:
+
+* an ssh tunnel
+* a network inspection tool such as [Fiddler](https://telerik.com/fiddler)
+* a corporate proxy
+
+Timing attacks might also be possible, though no concrete scenario has yet been
+laid out.
+
+See [#41](https://github.com/WICG/private-network-access/issues/41) for a discussion of these points.
+
+## 2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses? 
+
+Yes, apart from the above.
+
+## 3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+
+It does not.
+
+## 4. How do the features in your specification deal with sensitive information?
+
+It does not.
+
+## 5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+Yes, the permission status which allows a certain public webpage to access a certain private/local service.
+
+## 6. Do the features in your specification expose information about the underlying platform to origins?
+
+See question 1.
+
+## 7. Does this specification allow an origin to send data to the underlying platform?
+
+No.
+
+## 8. Do features in this specification enable access to device sensors?
+
+No.
+
+## 9. Do features in this specification enable new script execution/loading mechanisms?
+
+No.
+
+## 10. Do features in this specification allow an origin to access other devices?
+
+No, instead it restricts the ability of origins to access local network devices.
```

Comment:
Fair point, we should be more nuanced here. This explainer opens up access to devices from new contexts, as compared against a baseline of the previous version of Private Network Access. However, as compared to the current state of the web, it mostly introduces new hurdles (a permission prompt) instead of granting new capabilities. I say mostly, because it does grant new capabilities to secure contexts. These, however, are removed at the same time from non-secure contexts, which is an overall win: the capability has become restricted to secure contexts.

Reply:
Changed the statements.